pubspec.yaml

```diff
@@ -13,34 +13,18 @@ environment:
 
 dependencies:
   collection: ^1.15.0
-  connectivity_plus: ^5.0.2
+  connectivity_plus: ^6.1.1
   dio: ^5.0.0
   flutter:
     sdk: flutter
   gql: ^1.0.0+1
   graphql: ^5.1.3
   graphql_flutter: ^5.1.2
-  provider: ^6.0.5
+  provider: any
```

Comment:
Please specify the version.

Reply:
I edited the provider package to `any` to be more compatible with the user when using different versions of the `provider` or `Bloc` package on his project.